src/json/get-meta-from-path.js

```diff
@@ -1,11 +1,28 @@
-import parse, { getMeta, getValue } from './parse';
-
-export default function getMetaFromPath(jsonString, dataPath) {
+export default function getMetaFromPath(
+  jsonAst,
+  dataPath,
+  isIdentifierLocation
+) {
   // TODO: Handle json pointer escape notation and better error handling
-  return getMeta(
-    dataPath
-      .split('/')
-      .slice(1)
-      .reduce((obj, pointer) => getValue(obj, pointer), parse(jsonString))
-  );
+  const pointers = dataPath.split('/').slice(1);
+  const lastPointerIndex = pointers.length - 1;
+  return pointers.reduce((obj, pointer, idx) => {
```

Comment:
Interesting... Didn't think of using `reduce` to travers through a tree before.. Learned.

Reply:
It looks clean 😄

Reply:
Yeah, love it!